### ISSUE TITLE
effect doesn't work for terminal

### ISSUE THREAD

**Issue by AmrBinBashir** [EYHN/vscode-vibrancy#1]
The integrated terminal doesn't have the effect. 
![Annotation](https://user-images.githubusercontent.com/48618675/61163777-a8b4ed00-a510-11e9-90d1-a2a8bd4c0bf6.png)

**Comment by EYHN** [comment]
Thanks for feedback. 

In vscode, terminal is rendered through the canvas, and I can't make the background of the canvas transparent. I will reply here if there is any progress.

**Comment by AmrBinBashir** [comment]
> Thanks for feedback.
> 
> In vscode, terminal is rendered through the canvas, and I can't make the background of the canvas transparent. I will reply here if there is any progress.

you did an amazing job, I can live with terminal like this.

**Comment by syfxlin** [comment]
You should change the renderer type of the terminal to dom

`"terminal.integrated.rendererType": "dom"`

**Comment by AmrBinBashir** [comment]
@syfxlin 
I add this to my `settings.json` but no luck yet. Also tried to add `"terminal.background": "#00000000" `and still nothing.

**Comment by syfxlin** [comment]
@AmrBinBashir You need to modify the style of VSCode.
```css
.monaco-workbench .panel.integrated-terminal *:not(.xterm-cursor):not(.xterm-cursor-block) {
    background: transparent !important;
}
.monaco-workbench .panel.integrated-terminal *:hover:not(.xterm-cursor):not(.xterm-cursor-block) {
    background: transparent !important;
}
.monaco-workbench .panel.integrated-terminal .find-focused .xterm .xterm-viewport, .monaco-workbench .panel.integrated-terminal .xterm.focus .xterm-viewport, .monaco-workbench .panel.integrated-terminal .xterm:focus .xterm-viewport, .monaco-workbench .panel.integrated-terminal .xterm:hover .xterm-viewport {
    background: transparent !important;
}
.monaco-workbench .panel.integrated-terminal .terminal-outer-container {
    background-color: rgba(37, 37, 37,0.4);
}
```

**Comment by AmrBinBashir** [comment]
@syfxlin Thank you, It's working fine now. 
@EYHN I guess you can implement syfxlin code in the next version. Consider this issue solved.

**Comment by EYHN** [comment]
Thank you! @syfxlin 
I optimized this CSS. It has been released in 1.0.5.
